policyengine_us/variables/gov/irs/credits/retirement_savings/savers_qualified_contributions.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_us.model_api import *
+
+
+class savers_qualified_contributions(Variable):
+    value_type = float
+    entity = Person
+    label = "Retirement Savings Credit qualified contributions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.law.cornell.edu/uscode/text/26/25B#d_2"
+    adds = "gov.irs.credits.retirement_saving.qualified_retirement_savings_contributions"
```

Comment:
I think we should make this a full formula since it might be negative

Reply:
Just a simple:

    def formula(tax_unit, period, parameters):
        contributions = add(tax_unit, period, "gov.irs.credits.retirement_saving.qualified_retirement_savings_contributions")
        distributions = add(tax_unit, period, ["retirement_distributions'])
        return max_(contributions - distributions, 0)